src/ducks/transactions/TransactionModal.jsx

```diff
@@ -39,34 +41,67 @@ const showComingSoon = t => {
 const Separator = () => <hr className={styles.TransactionModalSeparator} />
 
 export const TransactionModalRow = ({
+  children,
   iconLeft,
   iconRight,
   disabled = false,
-  text,
+  className,
   ...props
 }) => (
   <Media
-    className={cx(styles.TransactionModalRow, {
-      [styles['TransactionModalRow-disabled']]: disabled
-    })}
+    className={cx(
+      styles.TransactionModalRow,
+      {
+        [styles['TransactionModalRow-disabled']]: disabled
+      },
+      className
+    )}
     {...props}
   >
     <Img className={styles.TransactionModalRowIcon}>
       {Icon.isProperIcon(iconLeft) ? (
-        <Icon icon={iconLeft} width={16} color={palette.slateGrey} />
+        <Icon
+          icon={iconLeft}
+          width={16}
+          color={palette.slateGrey}
+          className={cx({
+            [styles['TransactionModalRowIcon-alignTop']]: props.align === 'top'
```

Comment:
Why is it necessary to do this ? [Media](https://cozy.github.io/cozy-ui/react/#!/Media) align='top' is not adapted ?